x-pack/plugins/maps/public/classes/layers/vector_layer/utils.tsx

```diff
@@ -69,13 +69,15 @@ export async function syncVectorSource({
   } = syncContext;
   const dataRequestId = SOURCE_DATA_REQUEST_ID;
   const requestToken = Symbol(`${layerId}-${dataRequestId}`);
-  const canSkipFetch = await canSkipSourceUpdate({
-    source,
-    prevDataRequest,
-    nextMeta: requestMeta,
-    extentAware: source.isFilterByMapBounds(),
-  });
-  if (canSkipFetch) {
+  const doRefresh = syncContext.forceRefresh
+    ? syncContext.forceRefresh
```

Comment:
the conditional is on syncContext.forceRefresh so maybe just set this case to "true" to be explicit.